recipe/meta.yaml

```diff
@@ -29,9 +29,8 @@ requirements:
     - mpc
     - fmt
     # NOTE: gmp/mpir, mpfr, mpc, libflint, arb and fmt all use run_exports.
```

Comment:
That comment is now out-of-date, as boost now finally has a run-export as well. :)